client/src/components/CreatePost/Form/Form.js

```diff
@@ -58,16 +62,30 @@ const ModalComponent = ({ setCurrentStep, onClose }) => {
       setErrors(newErrors);
     }
   };
+  const cleanFormData = (formData) => {
+    let payload = {};
+    payload["title"] = formData.title;
+    payload["content"] = formData.description;
+    payload["expireAt"] = formData.expires;
+    payload["objective"] = formData.help;
+    payload["types"] = formData.tags;
+    payload["visibility"] = formData.shareWith;
+
+    payload["expireAt"] = payload["expireAt"].toLowerCase();
```

Comment:
Is it not possible to do `formData.expires.toLowerCase()` and `formData.shareWith.toLowerCase()`? 

I think it should allow you to do that since `expires` and `shareWith` seem to be strings...

Reply:
yeah here it's better to use dot notation
bracket notation is usually used for dynamic keys `obj[variableName]` or used to be more for characters that aren't allowed in JS syntax but nowadays with modern ECMASCript those are very rare cases I think, maybe for example built-in keywords such as `"class"`